source/read-write-concern/read-write-concern.rst

```diff
@@ -313,6 +313,33 @@ With MaxWireVersion < 4, ``writeConcern`` MUST be omitted when sending ``findAnd
 .. note ::
     Driver documentation SHOULD include a warning in their server 3.2 compatible releases that an elevated ``WriteConcern`` may cause performance degradation when using ``findAndModify``. This is because ``findAndModify`` will now be honoring a potentially high latency setting where it did not before.
 
+Helpers for other commands that write
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your driver has helpers for commands that write other than those discussed above, they SHOULD accept a ``writeConcern`` parameter.
```

Comment:
I think you should follow the lead of the existing spec here. Don't mention helpers here, just the commands that support writeConcern. You should also mention that passing writeConcern to any of these commands requires MongoDB 3.4+.

Finally, update the section on Write Concern under Test Plan (further into the spec) to add a note about testing that any of the relevant helpers don't send writeConcern if maxWireVersion < 5. See .4 about findAndModify as an example.